src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -62,6 +62,13 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('no_push', help="Indicates whether the image built should be pushed to the registry.", arg_type=get_three_state_flag())
         c.argument('no_logs', help="Do not show logs after successfully queuing the build.", action='store_true')
 
+    with self.argument_context('acr import') as c:
+        c.argument('source', help="A source identifier, either fully qualified (e.g. 'registry.azurecr.io/repository:tag') or partially qualified (e.g. 'repository@sha').")
+        c.argument('source_registry', options_list=['--registry', '-r'], help='The container registry of the source. It can be name, canonical name (login server) or resource ID of the source registry.')
+        c.argument('target_tags', options_list=['--target-tags', '-t'], help="The repository and optionally a tag in the 'repository:tag' format for target images.", action='append')
```

Comment:
--image?

Reply:
1. It is shortened as '-t'
2. As the variable itself implies "target_tags"